test/e2e/infinispan/smoke_test.go

```diff
@@ -65,7 +65,7 @@ func TestBaseFunctionality(t *testing.T) {
 	verifyNoPVCs(assert, ispn)
 	verifyLabelsAndAnnotations(assert, require, ispn)
 	verifyDefaultAuthention(require, ispn)
-	verifyScheduling(assert, require, ispn)
+	verifyScheduling(assert, require, ispn, 60)
```

Comment:
The test is failing because you're expecting a non-default value of 60 here, but you have not defined this in the Infinispan CR. You need to explicitly define 60 in the `tutils.DefaultSpec` function that starts on L26.